tracker/model.go

```diff
@@ -590,3 +591,38 @@ func clamp(a, min, max int) int {
 	}
 	return a
 }
+
+func (m *Model) CollectSendsTo(param Parameter) iter.Seq[sointu.Unit] {
```

Comment:
This probably should not yield sointu.Unit as this will return the entire Parameters map for the GUI to fool around. Rather, returning a struct with whatever data you need for visualizing the send e.g. something like `struct { ID int; Amount int; SourceInstrName string ... }` would make sense here. This way we don't unnecessarily expose the Unit.Parameters map to the GUI for modification.

Reply:
Also, eventually, if we go with the cache approach, this function could be very cheap: just essentially pulling the values from the cache, which gets updated only when the patch changes.

Reply:
implemented that now